web/Services/CsvImportService.cs

```diff
@@ -88,84 +94,143 @@ await using (var transaction = await _dbContext.Database.BeginTransactionAsync(I
-                    dbTenant.Phone = tenant.Phone;
-                    dbTenant.Postcode = tenant.Postcode;
-                    dbTenant.Type = tenant.Type;
-
-                    if (Boolean.TryParse(tenant.HasPet, out bool boolHasPet)|| tenant.HasPet is null)
-                    {
-                        dbTenant.HasPet = boolHasPet;
-                    }
-                    else
-                    {
-                        flashTypes.Add("danger");
-                        flashMessages.Add(InvalidInputMessage(tenant.Name, "HasPet", tenant.HasPet));
-                    }
-                    
-                    if (Boolean.TryParse(tenant.ETT, out bool boolETT)|| tenant.ETT is null)
-                    {
-                        dbTenant.ETT = boolETT;
-                    }
-                    else
-                    {
-                        flashTypes.Add("danger");
-                        flashMessages.Add(InvalidInputMessage(tenant.Name, "ETT", tenant.ETT));
-                    }
-                    
-                    if (Boolean.TryParse(tenant.UniversalCredit, out bool boolUniversalCredit)|| tenant.UniversalCredit is null)
-                    {
-                        dbTenant.UniversalCredit = boolUniversalCredit;
-                    }
-                    else
-                    {
-                        flashTypes.Add("danger");
-                        flashMessages.Add(InvalidInputMessage(tenant.Name, "UniversalCredit", tenant.UniversalCredit));
-                    }
-                    
-                    if (Boolean.TryParse(tenant.HousingBenefits, out bool boolHousingBenefits)|| tenant.HousingBenefits is null)
-                    {
-                        dbTenant.HousingBenefits = boolHousingBenefits;
-                    }
-                    else
-                    {
-                        flashTypes.Add("danger");
-                        flashMessages.Add(InvalidInputMessage(tenant.Name, "HousingBenefits", tenant.HousingBenefits));
-                    }
-                    
-                    if (Boolean.TryParse(tenant.Over35, out bool boolOver35)|| tenant.Over35 is null)
-                    {
-                        dbTenant.Over35 = boolOver35;
-                    }
-                    else
-                    {
-                        flashTypes.Add("danger");
-                        flashMessages.Add(InvalidInputMessage(tenant.Name, "Over35", tenant.Over35));
-                    }
-
-                    _dbContext.Tenants.Add(dbTenant);
+                    flashTypes.Add("danger");
+                    flashMessages.Add(InvalidInputMessage(tenant.Name, "Postcode", tenant.Postcode!));
                 }
-                _dbContext.SaveChanges();
             }
-            await transaction.CommitAsync();
+
+            dbTenant.Type = tenant.Type;
+
+            if (Boolean.TryParse(tenant.HasPet, out bool boolHasPet) || tenant.HasPet is null)

```

Comment:
You can do similar to this below as well

```suggestion
            if (bool.TryParse(tenant.HasPet, out var boolHasPet) || tenant.HasPet is null)

```